.drone/drone.yml

```diff
@@ -274,6 +274,37 @@ trigger:
   - refs/heads/r?
   - refs/heads/r??
 ---
+depends_on: []
+kind: pipeline
+name: release
+platform:
+  arch: amd64
+  os: linux
+steps:
+- commands:
+  - go install github.com/google/go-jsonnet/cmd/jsonnet@latest
+  - make release-snapshot
+  image: golang:1.18
+  name: test release
+  when:
```

Comment:
Were we running the release snapshot before? If we weren't I'd prefer removing it instead of running it on every PR.

Reply:
I'm setting up the stage to build OS package (deb and rpm). This will involve GPG signing and probably some OS-specific scripts. I think it would be best to have some kind of test as it would be a bit brittle/hard to maintain otherwise

As nfpm, the packaging tool, is integrated in goreleaser, the test I'm planning is:
1. Build snapshot release (creating the packages)
2. Install the packages on bare debian and centos/fedora images

Reply:
Good call :+1: